app/lib/features/chat_ui_showcase/models/convo_showcase_data.dart

```diff
@@ -313,3 +313,434 @@ final davidDm = createMockChatItem(
     ),
   ),
 );
+
+final imageMessageDmRoom16 = createMockChatItem(
+  roomId: 'mock-room-16',
+  displayName: 'David Chen',
+  activeMembersIds: ['@david:acter.global', '@kumarpalsinh:acter.global'],
+  isDm: true,
+  timelineEventItem: MockTimelineEventItem(
+    mockSenderId: '@david:acter.global',
+    mockOriginServerTs: 1743318966000, // March 30, 2025
+    mockMsgType: 'm.image',
+    mockEventType: 'm.room.message',
+    mockMsgContent: MockMsgContent(
+      mockBody: 'Image message about the API changes',
+    ),
```

Comment:
as said at the images below, we usually don't have two DMs with the same person, maybe we just a different username and id here?

```suggestion
  displayName: 'Meeko'
  activeMembersIds: ['@meeko:acter.global', '@kumarpalsinh:acter.global'],
  isDm: true,
  timelineEventItem: MockTimelineEventItem(
    mockSenderId: '@meeko:acter.global',
    mockOriginServerTs: 1743318966000, // March 30, 2025
    mockMsgType: 'm.image',
    mockEventType: 'm.room.message',
    mockMsgContent: MockMsgContent(
      mockBody: 'Image message about the API changes',
    ),
```

Reply:
Fixed by https://github.com/acterglobal/a3/pull/2834/commits/01484762db43a23a15909a85a30bbca97a9773b1